static/app/views/starfish/views/spanSummaryPage/spanBaselineTable.tsx

```diff
@@ -98,18 +89,18 @@ function BodyCell({
   }
 
   if (column.key === 'p95(span.self_time)') {
-    return <DurationCell milliseconds={row.metrics.p95} />;
+    return <DurationCell milliseconds={row.metrics?.['p95(span.duration)']} />;
   }
 
-  if (column.key === 'spans_per_second') {
-    return <ThroughputCell throughputPerSecond={row.metrics.spans_per_second} />;
+  if (column.key === 'spm()') {
+    return <ThroughputCell throughputPerSecond={row.metrics?.['spm()']} />;
```

Comment:
```suggestion
    return <ThroughputCell throughputPerSecond={row.metrics?.['spm()']/60} />;
```
We should divide this by 60 to convert spans per minute to spans per second.